### ISSUE TITLE
Automatically shutdown idle GPU containers in Kubeflow

### ISSUE THREAD

**Issue by supertetelman** [NVIDIA/deepops#667]
It is a common request to automatically shutdown Notebook containers if:

1. They have been running for a certain period of time (7 days)
2. They have been running idle for a certain period of time (4 hours)
3. They have had an idle GPU for a certain period of time (4 hours)

In order to conserve limited GPU resources it would be beneficial to automate shutdown of any Jupyter Notebook container or Pipeline container that meets one of these thresholds.

A lot of this functionality is/should be built into Kubeflow and it would be beneficical to point to the relevant docs, provide an example, and/or set reasonable configuration defaults in our deployment scripts.

**Comment by supertetelman** [comment]
I have created a branch that starts to address some of this.

I created a new directory under `workloads/services` that contains the Kubeflow overlay configs and copy over this configuration before running the `kfctl apply` command.

The current example I have provided keeps these feature off by default and address use case 2 above. It does not set a maximum time period nor does it specifically inspect for GPU usage.

Anyone looking for an example could probably take this and run with it.

Example config commit: https://github.com/supertetelman/deepops/commit/dd635c7d5ea3627e0f46046eee5d118b85d24435
My branch: https://github.com/supertetelman/deepops/tree/kubeflow-culling
Kubeflow PR: https://github.com/kubeflow/kubeflow/pull/3856
